test/e2e/flowaggregator_test.go

```diff
@@ -574,6 +569,32 @@ func checkRecordsForFlows(t *testing.T, data *TestData, srcIP string, dstIP stri
 	assert.GreaterOrEqualf(t, dataRecordsCount, expectedNumDataRecords, "IPFIX collector should receive expected number of flow records. Considered records: ", len(recordSlices))
 }
 
+func checkRecordsForToExternalFlows(t *testing.T, data *TestData, srcPodName string, srcIP string, dstIP string, dstPort int32, isIPv6 bool) {
```

Comment:
future improvement: I think it would make more sense to pass the `srcNodeName` as an argument to this function, instead of assuming it is `nodeName(0)`